src/app/board/quest/quest.component.ts

```diff
@@ -8,4 +9,13 @@ import { IQuest } from '../../shared/quest';
 })
 export class QuestComponent {
   @Input() quest: IQuest;
+
+  constructor(
+    private questService: QuestService
+  ) { }
+
+  public accept(): void {
+    this.questService.acceptQuest(this.quest.id)
+      .subscribe(() => undefined);
```

Comment:
If you don't need subscribe block in your components, you can make subscription in your service